Bottom: 273568c6e29ca10ab79e9df60a27396e3f5ba96d
Top:    ddfb7e42932e32ff6f79f0e7bd4d6ab292d5bc3d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-14 01:28:28 +0100

Refresh of gtk-gl-remove-unused-gc-erase-

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 8130f15..48d9fc2 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -51,7 +51,6 @@ typedef struct hid_gc_struct
   gint width;
   gint cap, join;
   gchar xor_mask;
-  gchar erase;
   gint mask_seq;
 }
 hid_gc_struct;
@@ -381,12 +380,10 @@ ghid_set_color (hidGC gc, const char *name)
   if (strcmp (name, "erase") == 0)
     {
       gdk_gc_set_foreground (gc->gc, &gport->bg_color);
-      gc->erase = 1;
     }
   else if (strcmp (name, "drill") == 0)
     {
       gdk_gc_set_foreground (gc->gc, &gport->offlimits_color);
-      gc->erase = 0;
     }
   else
     {
@@ -425,8 +422,6 @@ ghid_set_color (hidGC gc, const char *name)
 	{
 	  gdk_gc_set_foreground (gc->gc, &cc->color);
 	}
-
-      gc->erase = 0;
     }
 }
